src/utils/credentials-helper.js

```diff
@@ -0,0 +1,244 @@
+import {
+  getSolidDataset,
+  getThing,
+  createSolidDataset,
+  buildThing,
+  createThing,
+  saveSolidDatasetAt,
+  saveSolidDatasetInContainer,
+  setThing,
+  getStringNoLocale
+} from '@inrupt/solid-client';
+import { Session } from '@inrupt/solid-client-authn-browser';
+
+import { createDocAclForUser } from './session-helper';
+
+import { RDF_PREDICATES } from '../constants';
+
+const generateRsaKeyStrings = async () => {
+  const { privateKey, publicKey } = await window.crypto.subtle.generateKey(
+    {
+      name: 'RSA-PSS',
```

Comment:
`RSA-PSS` seems to be the easiest to use signing algorithm available through the `WebCrypto` API. I'm not against using a different algorithm if someone knows more than me.